apps/main/src/lend/lib/apiLending.ts

```diff
@@ -360,6 +360,9 @@ const user = {
     const results: { [userActiveKey: string]: UserLoanDetails } = {}
     const { signerAddress } = api
 
+    // Check if we're in beta mode to skip old bands data fetching
+    const isBeta = typeof window !== 'undefined' && localStorage.getItem('release-channel') === 'Beta'
```

Comment:
Please never hardcode the local storage keys - and let's use feature flags not `isBeta` directly
https://github.com/curvefi/curve-frontend/blob/4f887c9d28dbb373e6f4ac1e6ab426c263637958/packages/curve-ui-kit/src/hooks/useLocalStorage.ts#L38